protocol/codec_tester.go

```diff
@@ -230,7 +229,12 @@ func randomizeValue(v reflect.Value, datapath string, tag string, remainingChang
 
 	switch v.Kind() {
 	case reflect.Uint, reflect.Uintptr, reflect.Uint8, reflect.Uint16, reflect.Uint32, reflect.Uint64:
-		v.SetUint(rand.Uint64())
+		if strings.HasSuffix(datapath, "/HashType") {
+			// generate random value that will avoid protocol.ErrInvalidObject from HashType.Validate()
+			v.SetUint(rand.Uint64() % 3)
```

Comment:
can you use `MaxHashType` instead of the 3?

Reply:
It would lead to a circular import unfortunately — if MaxHashType were defined in the protocol package, it would work